src/misc/lv_color.h

```diff
@@ -110,6 +111,11 @@ typedef struct {
     uint8_t v;
 } lv_color_hsv_t;
 
+typedef struct {
+    uint8_t lumi;
+    uint8_t alpha;
+} lv_color16a_t;
```

Comment:
Should it be `lv_color8a_t`?

Reply:
`lv_color16a_t` is really misleading as `lv_color16_t` is RGB565. `lv_color_al88_t` would be cleanest, but it's different style. `lv_color8a_t` can be good trade off.

Reply:
While I agree that lv_color16a_t is not the best name, the naming scheme was that the number in the color format name indicates the number of bits of the value. Therefore I think color8a is a bit misleading, too, because it suggest that it is a 8 bit value, which it is not. Maybe lv_color8a8_t would be better (although ARGB8888 is called color32, and not something like color24a or color24a8).
Certainly, the cleanest solution would be to call them as the corresponding color format is called, i.e.:
lv_color32_t -> lv_argb8888_t
lv_color16_t -> lv_rgb565_t
lv_color16a_t -> lv_al88_t
But this would likely break too much code.

Reply:
Actually we can rename these by adding `typedef lv_color_argb8888_t lv_color32_t` in `lv_api_map_v9.h`.

Reply:
> lv_color32_t -> lv_argb8888_t
lv_color16_t -> lv_rgb565_t
lv_color16a_t -> lv_al88_t

I agree with it.  But it should be `lv_color16_t --> lv_al88_t` where 16 means it takes 16bits. There will be `lv_color[32|24|16|8]_t.`


> Actually we can rename these by adding typedef lv_color_argb8888_t lv_color32_t in lv_api_map_v9.h.

Let's go for it. It can be a separate PR. We can merge this massive achievement firstly.